client/my-sites/stats/activity-log/index.jsx

```diff
@@ -95,6 +95,34 @@ class ActivityLog extends Component {
 		window.scrollTo( 0, 0 );
 	}
 
+	getStartMoment() {
```

Comment:
Have you thought of separating this one in a separate utility like `adjustMoment`? Some tests could help making sure it works as expected.

Reply:
I agree, `adjustMoment` needs tests (see #17320 and the known issue with gmt offsets).

The hard part is parsing with an offset, and I think once I get that right in #17320, this method should become relatively simple. It's also _very_ tied to this component and its props, it doesn't feel quite right to extract it.